-- Error: tests/neg/annot-invalid.scala:9:21 ---------------------------------------------------------------------------
9 |  val x1: Int @annot(new Object {}) = 0 // error
  |                     ^^^^^^^^^^^^^
  |                     Expression cannot be used inside an annotation argument.
  |                     Tree: final class $anon() extends Object() {}
  |                     Type: Object {...}
-- Error: tests/neg/annot-invalid.scala:10:28 --------------------------------------------------------------------------
10 |  val x2: Int @annot({class C}) = 0 // error
   |                      ^^^^^^^
   |                      Expression cannot be used inside an annotation argument.
   |                      Tree: class C() extends Object() {}
   |                      Type: C
-- Error: tests/neg/annot-invalid.scala:11:26 --------------------------------------------------------------------------
11 |  val x3: Int @annot({val y: Int = 2}) = 0 // error
   |                      ^^^^^^^^^^^^^^
   |                      Expression cannot be used inside an annotation argument.
   |                      Tree: val y: Int = 2
   |                      Type: (y : Int)
-- Error: tests/neg/annot-invalid.scala:12:26 --------------------------------------------------------------------------
12 |  val x4: Int @annot({def f = 2}) = 0 // error
   |                      ^^^^^^^^^
   |                      Expression cannot be used inside an annotation argument.
   |                      Tree: def f: Int = 2
   |                      Type: (f : => Int)
-- Error: tests/neg/annot-invalid.scala:14:25 --------------------------------------------------------------------------
14 |    val x5: Int @annot('{4}) = 0 // error
   |                         ^
   |                         Expression cannot be used inside an annotation argument.
   |                         Tree: '{4}
   |                         Type: (scala.quoted.Quotes) ?=> scala.quoted.Expr[Int]
-- Error: tests/neg/annot-invalid.scala:16:9 ---------------------------------------------------------------------------
16 |  @annot(new Object {}) val y1: Int = 0 // error
   |         ^^^^^^^^^^^^^
   |         Expression cannot be used inside an annotation argument.
   |         Tree: final class $anon() extends Object() {}
   |         Type: Object {...}
-- Error: tests/neg/annot-invalid.scala:17:16 --------------------------------------------------------------------------
17 |  @annot({class C}) val y2: Int = 0  // error
   |          ^^^^^^^
   |          Expression cannot be used inside an annotation argument.
   |          Tree: class C() extends Object() {}
   |          Type: C
-- Error: tests/neg/annot-invalid.scala:18:14 --------------------------------------------------------------------------
18 |  @annot({val y: Int = 2}) val y3: Int = 0 // error
   |          ^^^^^^^^^^^^^^
   |          Expression cannot be used inside an annotation argument.
   |          Tree: val y: Int = 2
   |          Type: (y : Int)
-- Error: tests/neg/annot-invalid.scala:19:14 --------------------------------------------------------------------------
19 |  @annot({def f = 2}) val y4: Int = 0 // error
   |          ^^^^^^^^^
   |          Expression cannot be used inside an annotation argument.
   |          Tree: def f: Int = 2
   |          Type: (f : => Int)
-- Error: tests/neg/annot-invalid.scala:21:13 --------------------------------------------------------------------------
21 |    @annot('{4}) val y5: Int = 0 // error
   |             ^
   |             Expression cannot be used inside an annotation argument.
   |             Tree: '{4}
   |             Type: (scala.quoted.Quotes) ?=> scala.quoted.Expr[Int]
